CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ We use _breaking :warning:_ to mark changes that are not backward compatible (re
 
 ### Fixed
 
+- [#3800](https://github.com/thanos-io/thanos/pull/3800) Cleaning comments.
```

Comment:
We don't need to add this here. We like to keep the change log minimal and user-focussed, meaning it highlights external changes that the user of the software can notice rather than every single internal change and cleanup